src/store/facade/GroceryStore.java

```diff
@@ -546,4 +546,24 @@ private Result reorderProduct(Product product) {
 		return result;
 	}
 
+	/**
+	 * Changes the price of a product
+	 * 
+	 * @param request carries the relevant product fields
+	 * @return a result code that represents the outcome
+	 */
+	public Result changePrice(Request request) {
+		Result result = new Result();
+		Product product = productsList.searchById(request.getProductId());
+		product.setCurrentPrice(request.getProductCurrentPrice());
+
+		if (product.getCurrentPrice() == request.getProductCurrentPrice()) {
+			result.setResultCode(Result.ACTION_SUCCESSFUL);
+			System.out.printf("Product: %s, New Price: %.2f", product.getName(), product.getCurrentPrice());
```

Comment:
Ricky, please be careful with this line (#562). The output to the console should be UI's job. GroceryStore is on the business side of the app. :)